tests/integrations/server/raft_client.rs

```diff
@@ -89,3 +89,211 @@ fn test_batch_raft_fallback() {
+    let mut raft_client = get_raft_client(&pool);
+    let counter = Arc::new(AtomicUsize::new(0));
+
+    // Try to bind the mock server on a TCP port, and then do test.
+    for i in 0..100 {
+        let kv_service = MockKv(MockKvForRaft(Arc::clone(&counter)));
+        let port = 50000 + i;
+        let mut mock_server = match mock_kv_service(kv_service, "localhost", port) {
+            Ok(s) => s,
+            Err(_) => continue,
+        };
+        println!("port is {}",port);
+        mock_server.start();
+
+        let addr = format!("localhost:{}", port);
+
+        (0..50).for_each(|_| {
+            raft_client.send(1, &addr, RaftMessage::new()).unwrap();
+           // if let Err(err) = raft_client.send(1, &addr, RaftMessage::new()).unwrap() {
+           //     println!("{:?}",err), 
+           // };
+            thread::sleep(time::Duration::from_millis(10));
+            raft_client.flush();
+        });
+
+        println!("counter {:?}",counter.load(Ordering::SeqCst));
+
+       // thread::sleep(time::Duration::from_millis(1000));
+        println!("drop the server");
+        drop(mock_server);
+        (50..100).for_each(|_| {
+             raft_client.send(1, &addr, RaftMessage::new()).unwrap();
+           // if let Err(err) = raft_client.send(1, &addr, RaftMessage::new()).unwrap() {
+           //     println!("{:?}",err),
+           // };
+            thread::sleep(time::Duration::from_millis(10));
+            raft_client.flush();
+        });
+
+        println!("counter {:?}",counter.load(Ordering::SeqCst));
+        thread::sleep(time::Duration::from_millis(1000));
+        println!("start server begin");
+        
+        for i in 0..100 {
+            let kv_service = MockKv(MockKvForRaft(Arc::clone(&counter)));
+            let port = 50000 + i;
+            let mut mock_server = match mock_kv_service(kv_service, "localhost", port) {
+                Ok(s) => s,
+                Err(_) => continue,
+            };
+            println!("port is {}",port);
+            mock_server.start();
+
+            let addr = format!("localhost:{}", port);
+            (100..150).for_each(| n | {
+                       raft_client.send(1, &addr, RaftMessage::new()).unwrap();
+                      // if let Err(err) = raft_client.send(1, &addr, RaftMessage::new()).unwrap() {
+                      //      println!("{:?}",err),
+                      // };
+                       thread::sleep(time::Duration::from_millis(10));
+                       raft_client.flush();
+                       println!(" {} counter {}",n,counter.load(Ordering::SeqCst));
+             });
+             println!("counter {:?}",counter.load(Ordering::SeqCst));
+
+             assert!(counter.load(Ordering::SeqCst) > 50);
+             break;
+        }
+        assert!(counter.load(Ordering::SeqCst) >98);
+        break;
+    }
+ pool.shutdown().wait().unwrap();
+}
+
+#[test]
+fn test_batch_raft_network_isolation() {
```

Comment:
The test case needs `iptables`, and `iptables` needs root privillege. So please remove it, we can consider add it in schrodinger.